### ISSUE TITLE
Get kicked off of freenode because of no SASL authentication

### ISSUE THREAD

**Issue by kopasetik** [flackr/circ#300]
I've been kicked off of Freenode because CIRC had no SASL authentication. Therefore, I want to add SASL authentication. Is there a documented way to do this?

**Comment by MACscr** [comment]
So what do we do? Im mostly using my chromebook through tethering on my cell phone and it appears that freenode requires sasl when I do this. I pretty much only use freenode too, so i have to figure out a solution. =(

**Comment by Techman-** [comment]
@MACscr You're free to use another client if you need to. [KiwiIRC](https://kiwiirc.com) is a good in-browser client if you're interested. There's also other web-based solutions like [IRCCloud](https://www.irccloud.com) that you can look at.

**Comment by MACscr** [comment]
Of course, but circ is nice and would be great to be able to continue to use for my chromebook.

- Mark

**Comment by r04r** [comment]
@cdrichards I'm using cloaks without SASL. Not sure what you mean, all you have to do is identify before joining any channels.

**Comment by r04r** [comment]
@cdrichards Not really, you can also use a server password. Which CIRC supports.

**Comment by jdloft** [comment]
@r04r You can use that but SASL is needed for auto connecting to channels (which CIRC does) and still use a cloak. See this [faq entry](https://freenode.net/faq.shtml#nocloakonjoin) on the subject.

**Comment by r04r** [comment]
@jdloft Incorrect. Did you try it? I also linked a faq entry which suggests this method, and I just tried it again and it works as expected. You connect, you are identified and a cloak is applied before CIRC auto-joins any channels.

**Comment by jdloft** [comment]
@r04r Hmm. I saw that and tried the /connect command but CIRC uses /server so I thought it wouldn't work the same way.

**Comment by SkiftCreative** [comment]
I was having this issue as well, when i was tethering from my phone using tmobile. After I started using a vpn it fixed the issue as well. So that could be another way to solve the issue. But most vpn services costs a few $/mo.
